src/images/maps/geom_adm0.R

```diff
@@ -56,15 +56,16 @@ geom_adm0 <- function(iso3, ...) {
 #' Asserts that all geometries in `...` is contained within `y`. Maps
 #' `assert_covered_by` across all of `...`. Returns `TRUE` if no geoms passed in.
 #'
+#' Checks a buffered version of `y`, buffered by `0.3` degrees.
+#'
 #' @param x_list All geometries to check
 #' @param y sf class POLYGON/MULTIPOLYGON to use to check if all x falls within
-#' @param dist distance in meters (default = 1000)
 #'
 #' @return TRUE if all of `x_list` fall within `y`.
 assert_covered_all <- function(x_list, y) {
   y_buff <- suppressWarnings(
     suppressMessages(
-      sf$st_buffer(x = y, dist = 0.1)
+      sf$st_buffer(x = y, dist = 0.3)
```

Comment:
Would this create additional historical alerts if we regenerated all of the old signals?

Reply:
No, because if something isn't covered, an error is generated, so we wouldn't have generated historical alerts if we had this problem before. Seems to be something we've only run into now with this naval battle. There have been other naval battles in the ACLED data but must be quite rare and not been included in a Signal yet (or only been within 0.1 degrees).

Reply:
Oh and to clarify, checking is done at the time we generate the map, which is after creating the signals. We do no filtering on the data for the signals, hence why we check later that the map makes sense.

Reply:
Ok gotcha! That makes sense.